WebIDL bindings are not being re-generated after switching branches

I added an item to `components/script/dom/webidls/` and implemented it. Now when I switch back to the master branch, it complains that the implementation of this new item is missing and fails the build.

To reproduce, 
1. Grab https://github.com/servo/servo/pull/20144/commits
2. `./mach build` should be successful
3. `git checkout master`
4. `./mach build` will fail now, because `GetTexParameter` is not implemented on master, only on the other branch

```
error[E0046]: not all trait items implemented, missing: `GetTexParameter`
    --> components/script/dom/webgl2renderingcontext.rs:80:1
     |
80   | impl WebGL2RenderingContextMethods for WebGL2RenderingContext {
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `GetTexParameter` in implementation
     |
    ::: /home/mkollaro/data/dev/servo/target/debug/build/script-842b2b116fc9e009/out/Bindings/WebGL2RenderingContextBinding.rs
     |
9952 |     unsafe fn GetTexParameter(&self, cx: *mut JSContext, target: u32, pname: u32) -> JSVal;
     |     --------------------------------------------------------------------------------------- `GetTexParameter` from trait
error: aborting due to previous error
error: Could not compile `script`.
```

--- a/components/script/CMakeLists.txt
+++ b/components/script/CMakeLists.txt
@@ -106,7 +106,7 @@ foreach(binding IN LISTS bindings)
                    .
                    Bindings/${binding}Binding
                    ${webidls_src}/${binding}.webidl
-    DEPENDS Bindings ${bindinggen_deps} ${webidls_src}/${binding}.webidl ParserResults
+    DEPENDS Bindings ${bindinggen_deps} ${webidls} ParserResults
     VERBATIM
     )
   add_custom_target(${binding} DEPENDS Bindings/${binding}Binding.rs)

